src/script/entity/message/CallingTimeoutMessage.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Wire
+ * Copyright (C) 2021 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {REASON as AVS_REASON} from '@wireapp/avs';
+import {SuperType} from '../../message/SuperType';
+import {Message} from './Message';
+
+/**
+ * Call leave timeout system message
```

Comment:
Comment can be removed. I can already read that it is the `CallingTimeoutMessage` by looking at the class name.

Reply:
done in https://github.com/wireapp/wire-webapp/pull/10709/commits/946987f9d36e4bdd767f7bea7dc67deaebdd9e7a